coreapi/v23.0/staking/api/address.go

```diff
@@ -1,7 +0,0 @@
-package api
-
-import (
-	original "github.com/oasisprotocol/oasis-core/go/staking/api"
-)
-
-type Address = original.Address
```

Comment:
hm github doesn't detect this as moved

Reply:
isn't it in though?

Reply:
Oh, ok even better :) I didn't double-check.